### ISSUE TITLE
To perform ablation study

### ISSUE THREAD

**Issue by jingyaa** [luogen1996/MCN#2]
Hi,

Your job is so great!

I perform the experiments by your introductions and achieve 62.8 accuracy for RES. When I perform the ablation study of segmentation, I directly set the https://github.com/luogen1996/MCN/blob/bee86dff1a23e58cae112073db0c33863544ba46/model/mcn_model.py#L519 to 
loss += xy_loss * 0+ wh_loss * 0+ confidence_loss * 0+seg_loss*seg_loss_weight+co_enegy_loss * 0.

I want to ask if it is correct to set the loss functions of loss_rec and loss_cem directly to zero?
